src/lwc/geFormRenderer/geFormRenderer.js

```diff
@@ -1004,10 +1004,19 @@ export default class GeFormRenderer extends NavigationMixin(LightningElement) {
      * @param event The paymentError event object
      */
     handleAsyncWidgetError(event) {
-        let errorResponse = isNotEmpty(event.error.message[1]) ? event.error.message[1] : null;
+        let errorMessage = this.CUSTOM_LABELS.commonUnknownError;
```

Comment:
I thiiiiiiink this makes sense but just wanted to tag @psadie-sfdo and @kenlewis-sfdo to make sure!

Reply:
thanks, it would be great if this method is double or triple checked that it is coded as expected.
The method would fail (in console, the Elevate card widget would just display the spinner) if the error message is not an array. I set tokenization timeout to be 1000. I did reproduce the same issue on `feature/RD2`, but have not tested if the error is reported when timeout is 1000 on `master`.

<img width="2033" alt="GETokenTimeoutErrorNotDisplayedOnValidCard_Main" src="https://user-images.githubusercontent.com/28743286/95930482-69142b00-0d7b-11eb-8540-becda36a0081.png">

Reply:
The same issue is reproducible on an org built off the `master`.
<img width="1852" alt="Screen Shot 2020-10-13 at 6 31 08 PM" src="https://user-images.githubusercontent.com/28743286/95933063-551ff780-0d82-11eb-9990-9e17dcd36e72.png">

Reply:
This change looks good to me!